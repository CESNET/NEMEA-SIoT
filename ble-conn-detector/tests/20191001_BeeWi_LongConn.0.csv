time TIMESTAMP,macaddr DEV_ADDR,uint8 ATYPE,int8 RSSI
2033-01-08T00:40:00.000,07:00:00:cc:66:ab,32,118
2019-09-25T10:29:34.382,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:34.486,6d:73:5a:f3:7b:98,0,-39
2019-09-25T10:29:34.586,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:34.688,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:34.795,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:34.895,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:35.107,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:35.311,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:35.621,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:35.725,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:35.836,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:35.935,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:36.046,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:36.153,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:36.260,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:36.364,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:36.576,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:36.682,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:36.786,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:36.993,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:37.100,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:37.204,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:37.311,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:37.413,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:37.520,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:37.626,6d:73:5a:f3:7b:98,0,-40
2019-09-25T10:29:37.832,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:37.941,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:38.046,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:38.151,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:38.256,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:38.365,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:38.473,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:38.580,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:38.687,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:29:38.794,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:39.005,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:39.113,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:39.215,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:39.321,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:39.423,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:39.636,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:39.841,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:39.947,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:40.050,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:40.155,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:40.253,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:40.355,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:40.461,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:40.571,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:40.680,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:40.788,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:40.887,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:40.991,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:29:41.091,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:29:41.195,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:41.297,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:41.402,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:41.504,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:41.606,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:41.712,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:29:41.821,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:42.034,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:29:42.137,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:29:42.241,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:42.345,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:29:42.455,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:42.560,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:29:42.670,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:29:42.772,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:24.189,6d:73:5a:f3:7b:98,0,-39
2019-09-25T10:31:24.291,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:24.395,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:24.502,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:24.711,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:24.816,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:24.917,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:25.022,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:31:25.132,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:25.234,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:25.342,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:31:25.450,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:25.660,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:25.768,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:25.870,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:26.077,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:26.287,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:26.395,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:26.506,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:31:26.612,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:26.719,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:26.822,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:26.926,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:27.029,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:27.134,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:27.239,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:27.340,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:27.544,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:27.647,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:27.754,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:27.965,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:28.181,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:31:28.290,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:28.391,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:28.498,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:28.605,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:28.712,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:28.815,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:29.027,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:29.125,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:29.236,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:29.337,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:29.548,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:29.650,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:29.856,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:29.962,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:30.064,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:30.282,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:30.389,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:30.495,6d:73:5a:f3:7b:98,0,-38
2019-09-25T10:31:30.597,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:30.699,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:30.805,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:31:30.909,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:31.018,6d:73:5a:f3:7b:98,0,-39
2019-09-25T10:31:31.129,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:31.236,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:31.339,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:31.438,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:31.541,6d:73:5a:f3:7b:98,0,-39
2019-09-25T10:31:31.754,6d:73:5a:f3:7b:98,0,-37
2019-09-25T10:31:31.854,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:31.956,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:32.061,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:31:32.169,6d:73:5a:f3:7b:98,0,-39
2019-09-25T10:31:32.274,6d:73:5a:f3:7b:98,0,-44
