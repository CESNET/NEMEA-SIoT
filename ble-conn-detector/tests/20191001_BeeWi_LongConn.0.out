2019-09-25T10:31:24.189,6d:73:5a:f3:7b:98,0,101417,0,"The device 6d:73:5a:f3:7b:98 was in use for 101417us."
